### PR TITLE
Quiet ComboBox design fixes

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes https://github.com/adobe/react-spectrum/issues/1444

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/label/src/Field.tsx [hunk 1]
```diff
@@ -54,6 +55,7 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
         className={labelWrapperClass}>
         <Label
           {...labelProps}
+          UNSAFE_className={labelClassName}
```
Comment (LFDanLu) [inline packages/@react-spectrum/label/src/Field.tsx:58]: hmm maybe I should make Field accept a isQuiet prop

File: packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css [hunk 2]
```diff
@@ -75,6 +75,14 @@ governing permissions and limitations under the License.
       width: 100%;
       flex: 1 1 auto;
     }
+
+    .spectrum-FieldLabel--quiet {
+      padding-bottom: var(--spectrum-fieldlabel-quiet-padding-bottom);
+      padding-top: var(--spectrum-fieldlabel-quiet-padding-top);
+      line-height: var(--spectrum-fieldlabel-quiet-text-line-height);
+      /* Negative margin so the label is positioned properly */
+      margin-bottom: -3px;
```
Comment (devongovett) [inline packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css:84]: Shouldn't this come from DNA?

File: packages/@react-spectrum/label/src/Field.tsx (hunk 1 above)
Comment (devongovett) [reply packages/@react-spectrum/label/src/Field.tsx:60]: That makes sense.

File: packages/@adobe/spectrum-css-temp/components/textfield/index.css [hunk 3]
```diff
@@ -179,6 +179,10 @@ governing permissions and limitations under the License.
     /* Treat all quiet inputs and textareas the same */
     resize: none;
     overflow-y: hidden;
+
+    padding-top: var(--spectrum-textfield-quiet-padding-top);
```
Comment (devongovett) [inline packages/@adobe/spectrum-css-temp/components/textfield/index.css:183]: This scares me. Why are we adding padding to textfield?

**Review by devongovett — COMMENTED** [review]
I'm not a huge fan of patching DNA for this. Are the variables not available already for it? Should we wait to update DNA?

File: packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css (hunk 2 above)
Comment (LFDanLu) [reply packages/@adobe/spectrum-css-temp/components/fieldlabel/index.css:84]: I did a search through `dna-vars` after building it locally and couldn't find anything that mentioned a `quiet` variant of the field label. In XD for the 5.3 textfield, the label for quiet textfield is aligned in such a way that the label text has 0 padding between it and the top of the input element, hence why I added these vars.

File: packages/@adobe/spectrum-css-temp/components/textfield/index.css (hunk 3 above)
Comment (LFDanLu) [reply packages/@adobe/spectrum-css-temp/components/textfield/index.css:183]: This overrides the padding set here: https://github.com/adobe/react-spectrum/blob/main/packages/@adobe/spectrum-css-temp/components/textfield/index.css#L55.
In the XD file for 5.3 textfields, quiet textfields have a padding bottom of 8px (9px -1px from the border) and padding top of 6px

**Comment by LFDanLu** [comment]
Pending update to DNA

**Comment by LFDanLu** [comment]
Closing for now, will reopen if we revisit

**Pull request closed without merging**